src/network/networkd-network.h

```diff
@@ -164,6 +164,7 @@ struct Network {
         bool allow_port_to_be_root;
         bool unicast_flood;
         unsigned cost;
+        unsigned priority;
```

Comment:
hmm, you define that as "unsigned" here (i.e. 32bit), but the netlink type appears to be 16bit only. Hence please use uint16_t here too, so that we use the same and correct type all the way